src/js/control-bar/text-track-controls/caption-settings-menu-item.js

```diff
@@ -53,6 +53,33 @@ class CaptionSettingsMenuItem extends TextTrackMenuItem {
    */
   handleClick(event) {
     this.player().getChild('textTrackSettings').open();
+
+    const spatialNavigation = this.player_.spatialNavigation;
+    const isSpatialNavListening = spatialNavigation && spatialNavigation.isListening_ && !spatialNavigation.isPaused_;
+
+    if (isSpatialNavListening) {
+      this.searchForTrackSelect();
+    }
+  }
+
+  /**
+   * This gets called by 'handleClick' if 'spatialNavigation' is enabled.
+   * Searches for the first 'TextTrackSelect' inside of modal to focus.
+   */
+  searchForTrackSelect() {
```

Comment:
@wseymour15 We moved this function to the spatial navigation in the last merged PR for the branch.

Reply:
Yeah, I still think this can be private if it only used in this class.

Reply:
> Yeah, I still think this can be private if it only used in this class.

@wseymour15 thanks for the approval I think I will merge the PR so we do not lose the approvals & do an small PR with that change